console/console.go

```diff
@@ -102,10 +100,6 @@ func GetConsolePluginCR(consolePort int, basePath string, serviceNamespace strin
 	}
 }
 
-func GetBasePath(clusterVersion string) string {
-	if strings.Contains(clusterVersion, "4.11") {
-		return compatibilityBasePath
```

Comment:
Maybe the declaration of var(line 31) should be removed as well?

Reply:
removed